runtime/jvmti/jvmtiHook.c

```diff
@@ -239,15 +221,23 @@ jvmtiHookThreadDestroy(J9HookInterface** hook, UDATA eventNum, void* eventData,
 	/* The final thread in the system is destroyed very late, JVMTI could be shut down by then */
```

Comment:
```suggestion
	/* The final thread in the system is destroyed very late, JVMTI could be shut down by then. */
```

Reply:
Isn't it better to leave unchanged code alone?

Reply:
You can. It is not a big deal. Almost the entire function is updated. So, I would have fixed the formatting for the function. Also, the same comment is repeated in `jvmtiHookVirtualThreadDestroy` with a full-stop (inconsistent).